cmd/status-cli/main.go

```diff
@@ -0,0 +1,324 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/ethereum/go-ethereum/crypto"
+
+	"go.uber.org/zap"
+
+	"github.com/google/uuid"
+
+	"github.com/status-im/status-go/account/generator"
+	"github.com/status-im/status-go/appdatabase"
+	"github.com/status-im/status-go/common/dbsetup"
+	gethbridge "github.com/status-im/status-go/eth-node/bridge/geth"
+	"github.com/status-im/status-go/eth-node/types"
+	"github.com/status-im/status-go/logutils"
+	"github.com/status-im/status-go/multiaccounts"
+	"github.com/status-im/status-go/protocol"
+	"github.com/status-im/status-go/protocol/common"
+	"github.com/status-im/status-go/protocol/protobuf"
+	"github.com/status-im/status-go/protocol/requests"
+	"github.com/status-im/status-go/t/helpers"
+	"github.com/status-im/status-go/wakuv2"
+	"github.com/status-im/status-go/walletdatabase"
+
+	"github.com/urfave/cli/v2"
+)
+
+const enrBootstrap = "enrtree://AMOJVZX4V6EXP7NTJPMAYJYST2QP6AJXYW76IU6VGJS7UVSNDYZG4@boot.test.shards.nodes.status.im"
```

Comment:
This variable is defined in `defaults` package. Let's reuse it from there 🙂 
https://github.com/status-im/status-go/blob/7816ee186d27acf30b2d3d7ef217cfec7416efd6/api/defaults.go#L38-L44